db-checkbook/README.md

```diff
@@ -0,0 +1,11 @@
+# Checkbook NYC Exploratory Work and Join to CPDB
+
+## To do:
+- upload relevant Checkbook NYC data to Digital Ocean
+- write the code to pull down data from Digital Ocean
+- Dea and Ali make their own branches and start working on smallest units of extractable code from our original notebooks, checking it into our projects, implementing, then submitting PR
+- mimic utility of methods from db-facilities/facdb/cli.py (init, dataloading, build, etc) using facdb terminology
+- make a data product-level requirements doc if needed, and update top-level requirements doc (requirements.in) in monorepo with python modules that are generally applicable to the team's work
+
```

Comment:
I suppose to Dea's question on chat, there's an additional step for uploading the data to s3 at the end.

Reply:
Which we would definitely want to include!